### ISSUE TITLE
Add configs to include custom activity bar view

### ISSUE THREAD

**Issue by rosinality** [Gruntfuggly/activitusbar#6]
Hi, thank you for nice extension!

As vscode 1.23 came with custom activity bar functionalities, I tried to make it configurable to include custom activity bar buttons in the activitusbar. https://github.com/rosinality/activitusbar/commit/03ceb83f028c027cd77bd77a3ebf28c755af719e

I think this can be useful for some use-cases. Could I contribute this?

**Comment by rosinality** [comment]
I tried to minimize modifications. Extends activitusbar.views to be able to include custom view commands, and add new configurations for icons to each commands. But as Commands extension that you suggested provides more general capabilities, maybe it is preferable to make it to be more general.

**Comment by rosinality** [comment]
I have implemented it and it works for my use-cases. (adding code outline button) Could I make pull requests?

**Comment by Gruntfuggly** [comment]
Sure. Can you post an example config too?

**Comment by rosinality** [comment]
Made https://github.com/Gruntfuggly/activitusbar/pull/7

Example config will be like this:

```json
"activitusbar.views": "explorer,search,scm,debug,extensions,extension.XXX",
"activitusbar.viewsIcons": {
    "explorer": "file-text",
    "search": "search",
    "scm": "repo-forked",
    "debug": "bug",
    "extensions": "package",
    "extension.XXX": "list-unordered"
}
```

extension.XXX is the id of activitybar of the extensions, because view command of the custom activity bar is in form like "workbench.view.extension.XXX". Maybe it can be cumbersome to get the id of the extensions.

**Comment by Gruntfuggly** [comment]
Hi - I've uploaded a new version including your changes. I've modified the configuration so that it maps the view names to the icons as I think this will be easier to understand and maintain. I've also made it so that it automatically migrates the old style configuration to the new, which unfortunately means you might lose your configuration - sorry! You'll just need to add 
"extension.outline" : "list-unordered" to *activitusbar.views*. Sorry for the inconvenience...
